### ISSUE TITLE
HF0.2, cli_wallet: buy command casts parameters wrongly

### ISSUE THREAD

**Issue by gropox** [GolosChain/golos#284]
does **buy** command interpretes parameters *rate* and *amount* wrongly?

 min_to_receive 0.001 GOLOS, but as parameter supplied "1". Tries different values like 1.000 or "1.000"  - result is the same. Probably again "asset initialization issue"?

```
unlocked >>> buy business "GOLOS" "GBG" 1 1 9 true

{
  "ref_block_num": 28594,
  "ref_block_prefix": 2123464866,
  "expiration": "2017-11-16T22:55:54",
  "operations": [[
      "limit_order_create",{
        "owner": "business",
        "order_id": 9,
        "amount_to_sell": "0.001 GBG",
        "min_to_receive": "0.001 GOLOS",
        "fill_or_kill": false,
        "expiration": "1969-12-31T23:59:59"
      }
    ]
  ],
  "extensions": [],
  "signatures": [
    "1f2aa8d17c8f6e7887e641e67fa5abe780f537ab84317f4b660ae67c3524519b2c54ddb612489058ea68b1fa65910b9b42d2b0f3915f05d2dd98936f3c1831b973"
  ]
}
```

**Comment by Nemo1369** [comment]
@gropox Thanks for the issue. Not really an "asset initialisation issue". Needs to be initialised through ```from_string```.

**Comment by Nemo1369** [comment]
Fixed with 50bc7bf0b9d2ab1f4bc2298600a589d6332badd9
